packages/grid/.gitignore

```diff
@@ -1,22 +1,9 @@
 
 # See https://help.github.com/ignore-files/ for more about ignoring files.
 
-# dependencies
-node_modules
-
 # builds
-build
-dist
 .rpt2_cache
 
 # misc
-.DS_Store
 .env
```

Comment:
Delete this. We shouldn't ignore the non-local `.env` files, but also there is no `.env` in grid. Only in the apps (code-studio, embed-grid, embed-chart)